connectivity/service/src/test/java/org/eclipse/ditto/connectivity/service/messaging/mqtt/hivemq/consuming/MqttConsumerActorTest.java

```diff
@@ -524,7 +506,7 @@ public void acknowledgeRejectedIncomingMessageIfShouldNotRedeliver() {
         final var onCompleteMessage = new Object();
         final var fakeMqttClientActor = ACTOR_SYSTEM_RESOURCE.newTestKit();
 
-        final var underTest = fakeMqttClientActor.watch(fakeMqttClientActor.childActorOf(
+        fakeMqttClientActor.watch(fakeMqttClientActor.childActorOf(
```

Comment:
Watching is superfluous.

Reply:
It's not because of this assertion: https://github.com/eclipse/ditto/pull/1435/files#diff-5fbc4686d5587b526f2959c16e6d951e499da06237027efe5424986624d6c4f2R532